### ISSUE TITLE
GSSoC: [FEATURE] : Add Analytics page

### ISSUE THREAD

**Issue by dhruv8433** [oxiton-foundation/click-metrics#71]
@oxiton-foundation

currently there are nothing in analytics page

![image](https://github.com/oxiton-foundation/click-metrics/assets/114583978/90c75258-c06e-4b3b-b12b-79c2bbe6ef40)

but we can redesign and it will look like 

![image](https://github.com/oxiton-foundation/click-metrics/assets/114583978/d4226bc9-254f-4e3b-9cc8-b564f3f01286)

**Comment by Swarnendu0123** [comment]
Discuss your approach clearly!

Prerequisites: #24

**Comment by dhruv8433** [comment]
@Swarnendu0123 

done: Prerequisites #24
in `analytics` page currently there is nothing

but we can create one analytics page at where user can show their history, usages, how many qr code they generated and if the limit is over than we can popup them for subscribe 

like wise we can create proper graaph based analytics page

**Comment by Swarnendu0123** [comment]
There is already the same pre-existing issue. Ask me to assign there. You can continue there.

**Comment by Swarnendu0123** [comment]
Please claim a new issue as this is a copy issue. and the original is already taken!
